README.md

```diff
@@ -1,29 +1,88 @@
 
-# nemos
-NEural MOdelS, a statistical modeling framework for neuroscience.
+`nemos` ("NEural MOdelS") is a statistical modeling framework for systems
+neuroscience, built on top of [jax](jax.readthedocs.io/). nemos aims to provide
+well-tested, GPU-accelerated implementations of standard statistical modeling
+methods. nemos is not attempting to provide all the latest and greatest methods,
+but instead to provide a stable place to start, which you can build off of or
+compare against. For now, we are focusing on the Generalized Linear Model (GLM);
+the package is under active development and more methods will be added in the
+future.
 
-## Disclaimer
-This is an alpha version, the code is in active development and the API is subject to change.
+To learn more about the Generalized Linear Model, we recommend [Neuromatch
+Academy's
+lesson](https://compneuro.neuromatch.io/tutorials/W1D3_GeneralizedLinearModels/student/W1D3_Intro.html)
+and [Jonathan Pillow's Cosyne 2018
+tutorial](https://www.youtube.com/watch?v=NFeGW5ljUoI&t=424s).
+
+## Basic usage
 
-## Setup
+The core object of nemos is the
+[`GLM`](https://nemos.readthedocs.io/en/latest/reference/nemos/glm/) object,
+which is built as an extension of scikit-learn's
+[estimator](https://scikit-learn.org/stable/modules/generated/sklearn.base.BaseEstimator.html#sklearn.base.BaseEstimator)
+object:
 
-To install, clone this repo and install using `pip`:
+```python
+import nemos as nmo
+# Create predictors X and targets y
+X = ...
+y = ...
 
-``` sh
-git clone git@github.com:flatironinstitute/nemos.git
-cd nemos/
-pip install -e .
+glm = nmo.glm.GLM()
+glm.fit(X, y)
+
+# Investigate GLM model parameters
+glm.coef_
+glm.intercept_
 ```
 
-If you have a GPU, you may need to install jax separately to get the proper
-build. The following has worked for me on a Flatiron Linux workstation: `conda
-install jax cuda-nvcc -c conda-forge -c nvidia`. Note this should be done
-without `jax` and `jaxlib` already installed, so either run this before the
-earlier `pip install` command or uninstall them first (`pip uninstall jax
-jaxlib`). See [jax docs](https://github.com/google/jax#conda-installation) for
-details (the `pip` instructions did not work for me).
+Nemos `GLM` objects predict spiking from a single neuron in response to
+user-specified predictors. The predictors `X` must be a 2d array with shape
+`(n_timebins, n_features)`, and `y` must be a 1d array with shape
+`(n_timebins,)`. We recommend using
+[pynapple](https://github.com/pynapple-org/pynapple) for initial exploration and
+reshaping of your data!
+
+When initializing the `GLM` object, users can optionally specify the
+[observation
+model](https://nemos.readthedocs.io/en/latest/reference/nemos/observation_models/)
+(also known as the noise model) and the
+[regularizer](https://nemos.readthedocs.io/en/latest/reference/nemos/regularizer/).
```

Comment:
Maybe list available regularizer ?

Reply:
I don't think so (because then we have to keep updating it as we change), if they're interested in which regularizers, they can follow the link